docs/editor_guide.md

```diff
@@ -94,8 +93,69 @@ these steps need to be taken:
+different local/domain communities.
+This helps to ensure that the lesson is accessible
+and guards against [expertise awareness gap](https://carpentries.github.io/instructor-training/04-expertise.html#mind-the-gap).
+
+So far, reviewers have been identified through a combination of:
+
+* responses to the Lab review volunteer form
+* the Editor's professional network
+* exploring the lists of 'watchers' on the repository of the lesson under review,
+  membership of any relevant Slack channels, etc
+
+Invitations to review have been made privately, by email/Slack message,
+followed by public invitation and confirmation on the review thread in this repository.
+(See examples of these posts in [the _Good Enough Practices_ lesson review thread](https://github.com/carpentries-lab/reviews/issues/24#issuecomment-1451796473).)
+
+Potential reviewers frequently ask what the time frame is for review.
+We do not have a strict policy,
+but hope to receive reviews within six weeks of confirmation.
+However, we would rather receive good reviews from reviewers who are a good fit for the lesson.
+So the Editor should encourage would-be reviewers to
+estimate a timeframe that would work for them
+and try to communicate that to the authors.
+
+#### Support for Editors from The Carpentries
+The [Curriculum Team](mailto:curriculum@carpentries.org) can provide a list of
+people who have volunteered to review lessons in the Lab,
+and can contact people on your behalf
+e.g. where you have identfied a potential reviewer from the community,
+whose details are stored in AMY.
+
+### Supporting the Review Process
+After assigning reviewers, the Editor should continue to support the process by:
+
+- answering questions from authors and reviewers
+- reaching out to reviewers/authors after a long period without response
+- summarising reviews when they come in,
+  e.g. to collate and re-state the requested changes for authors to respond to
+- providing guidance on which changes are within scope for a reviewer to request,
+  based on [the _Review Scope_ section of the Reviewer Guide](https://github.com/carpentries-lab/reviews/blob/main/docs/reviewer_guide.md#review-scope)
+  in combination with the Editor's own judgement.
+
+### Accepting lessons
+After authors have responded to reviewer's comments and requested changes,
+the Editor should ask the reviewers to evaluate the lesson again
+and respond (at least) once more, either
+to confirm that they are happy for the lesson to be accepted, or
+to make further comments/request additional changes.
+(See the _Good Enough Practices_ an example in lesson review thread](https://github.com/carpentries-lab/reviews/issues/24#issuecomment-1588926986).)
 
+### After acceptance
 1. If the author(s) have indicated that they would like to submit the lesson to JOSE,
   assist them with getting started on that process and ensure that the JOSE editors
   are aware that the lesson has already been accepted to The Carpentries Lab.
```

Comment:
this would be a step that the carpentries curriculum team could help with even if conflicted on a particular lesson?

Reply:
Yes, I can take back over as soon as the Lab review is complete.